project/UI/Jekyll/Jekyll.swift

```diff
@@ -21,8 +21,8 @@ public struct JekyllCommand: ParsableCommand {
     @ArgumentParser.Option(help: "Path where the resulting jekyll files will be generated")
     private var output: ArgumentPath
     
-    @ArgumentParser.Option(name: .customLong("main-page"), default: "README.md", help: "Path to 'README.md' file to be used as the index page")
-    private var mainPage: String
+    @ArgumentParser.Option(name: .customLong("main-page"), help: "Path to 'README.md' file to be used as the index page")
+    private var mainPage: String = "README.md"
```

Comment:
I don't think this should have a default value. If this option is not provided and the file `README.md` does not exist, what is the expected outcome?

Reply:
As Jekyll works using our sidebar template always needs a `main-page` - in case you set your main page, it will take it in another case it will create a new one called `README.md` with empty content.